cypress/integration/play-button.js

```diff
@@ -0,0 +1,21 @@
+describe('Stop and play the music', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000');
+  });
+  it('first test case', () => {
+    cy.get('#playContainer').click();
+    const expectPlayingAudio = () => {
+      cy.get('audio').should(elt => {
+        let audible = false;
+        elt.each((i, el) => {
+          if (el.duration > 0 && !el.paused && !el.muted) {
+            audible = true;
+          }
+        });
+        expect(audible).to.eq(true);
+      });
+    };
+    expectPlayingAudio();
```

Comment:
```suggestion
    cy.get('audio').should(audioElements => {
      const audioIsPaused = audioElements[0].paused;
      expect(audioIsPaused).to.eq(false);
    });
```

Reply:
Since we have only one audio element on the page, it might be better to simplify things and keep the arguments descriptive

Reply:
There is no need you think to test on the other properties of the audio element (muted, duration) ?